extensions/target-specific/chromium/blink/fragment-tree/fragment-tree.js

```diff
@@ -15,6 +15,18 @@ Loader.OnLoad(function() {
                         !box.isNull() ? box : !linebox.isNull() ? linebox : text);
     }
 
+    function layoutObjectChildrenToFragments(object) {
+        return object.array("child_objects_").dcast(new DbgObjectType("blink::LayoutBox", object.type)).map((box) => {
+            if (box.isNull())
+                return box;
+            return box.f("cached_layout_result_.ptr_.physical_fragment_.ptr_").then((fragment) => {

```

Comment:
Consider adding an extended field for the fragment of a blink::LayoutBox. You can call that here and it seems like something that might be useful elsewhere.

Reply:
Done